test/indexer/__init__.py

```diff
@@ -123,9 +120,9 @@ def load(version):
     def _load_canned_file_version(cls,
                                   *,
                                   uuid: str,
-                                  version: Optional[str],
+                                  version: str | None,
                                   extension: str
-                                  ) -> Union[MutableJSONs, MutableJSON]:
+                                  ) -> MutableJSONs | MutableJSON:
```

Comment:
```suggestion
                                  ) -> MutableJSON | MutableJSONs:
```

For consistency with the similar refactoring in `_load_canned_file()`